doc/usage.md

```diff
@@ -16,17 +16,17 @@ OPTIONS:
                                         easily (e.g. test and production 
                                         setup).Sample of configuration file 
                                         parameters looks like:
-                                        vss = vss_release_2.2.json
+                                        vss = vss_rel_3.0.json
```

Comment:
I think we should name the releases consistenly. Everything is called "release" and 3.0 only "rel". I suggest changin it to release as well.  Will not mark all instances of this

Reply:
My original idea for changing was to use the same naming as in VSS, i.e. same name as in https://github.com/COVESA/vehicle_signal_specification/releases/download/v3.0/vss_rel_3.0.json, and keep the name for the others to do not cause troubles for anyone referring to them.

But I have no problem using the existing name pattern.

Reply:
Let's use "release", looks nicer, and we do not need save bytes here